lib/dalli/protocol/meta.rb

```diff
@@ -68,25 +74,42 @@ def read_multi_req(keys)
           # VA value_length flags key
           tokens = line.split
           value = @connection_manager.read_exact(tokens[1].to_i)
+          bitflags = optimized_for_raw ? 0 : @response_processor.bitflags_from_tokens(tokens)
```

Comment:
since we didn't ask for the response to have fit flags they will not be there and will be 0, skip all the parsing. across the whole batch.